_episodes/47.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: episode
+title: "47: Meet the new Trino maintainers"
+date: 2023-03-15
+tags: database memory in-memory apache ignite
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Trino releases"
+  desc: "411-416"
+  time: 
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [James Petty](https://github.com/pettyjamesm), ___ at AWS
+* Also Manfred. Kind of.
+
+## Releases 411-414
+
+Official highlights from Martin Traverso:
+
+[Trino 411](https://trino.io/docs/current/release/release-411.html)
+
+* `migrate` procedure to convert a Hive table to Iceberg.
+* Join and `LIKE` pushdown in Ignite.
+* Support for `DELETE` in Ignite.
+* `procedure` table function for executing stored procedures in SQL Server.
+* Faster join queries over Hive bucketed tables.
+* Faster planning for tables with many columns in Hive.
+
+[Trino 412](https://trino.io/docs/current/release/release-412.html)
+
+* New `exclude_columns` table function.
+* Support for `ADD COLUMN` in Ignite.
+* Support for table comments in PostgreSQL connector.
+* Faster sum(`DISTINCT ...`) queries for various connectors.
+
+[Trino 413](https://trino.io/docs/current/release/release-413.html)
+
+* Support for `MERGE` in the Phoenix connector.
+* Support for table comments in the Oracle connector.
+* Improved performance of queries involving window functions or `MATCH_RECOGNIZE`.
+
+[Trino 414](https://trino.io/docs/current/release/release-413.html)
+
+* Experimental support for tracing using OpenTelemetry.
+* Support for Databricks 12.2 LTS in Delta Lake connector.
+* Support for fault-tolerant execution in Redshift connector.
+* `sequence` table function.
+
+## Introducing the two new Trino maintainers
+
+Manfred should hardly need an introduction to Trino Community broadcast viewers,
+as he's been around and hosting episodes for a while. But in the background, 
+he's also been quietly working on docs, the website, and a wide variety of other
+initiatives in the Trino.
+
+James should also be familiar to anyone who has contributed on Trino. Iconically
+rocking a GitHub avatar of the face of Bob Ross, it's hard to miss when he shows
+up on a pull request. And working on Trino as part of AWS Athena, he's been a
+major engineering contributor for the last several years, with 262 commits
+under his belt and more on the way.
+
+## What is a maintainer?
+
+If you don't go clicking around on the Trino website fanatically trying to find
+everything you can possibly read about the project, there's a chance you've
+never bumped into our [roles](https://trino.io/development/roles.html) page,
```

Comment:
use site.url .. test locally or on the preview to verify it stays on the same host